cpp/include/cudf/column/column_view.hpp

```diff
@@ -600,16 +648,22 @@ class mutable_column_view : public detail::column_view_base {
 
   /**
    * @brief Returns the number of child columns.
+   *
+   * @return The number of child columns
    */
   [[nodiscard]] size_type num_children() const noexcept { return mutable_children.size(); }
 
   /**
    * @brief Returns iterator to the beginning of the ordered sequence of child column-views.
+   *
+   * @return iterator to the beginning of the ordered sequence of child column-views
```

Comment:
Once you decide whether to accept my suggestions for the iterators above verbatim or modify them, I would copy whatever docstring you end up with to this `child_begin` docstring and the `child_end` docstring below.